v4l2-request-test.c

```diff
@@ -29,7 +29,7 @@
 #include <time.h>
 #include <unistd.h>
 
-#include <drm_fourcc.h>
+#include <drm/drm_fourcc.h>
```

Comment:
We're taking drm_fourcc.h from libdrm, not from the kernel headers. Our Makefile adds -I/usr/include/libdrm/ to our CFLAGS (using pkg-config), where the headers are located. So NAK.